lib/components/user/notification-prefs-pane.js

```diff
@@ -85,13 +223,60 @@ class NotificationPrefsPane extends Component {
             </FormGroup>
```

Comment:
In the ToggleButton group above, it might be preferable just to handle setting the notification channel (i.e., making an updateUser request to the backend) on click.

Reply:
Should the `Save Preferences` button be removed altogether if we are going to update the backend on the spot?

Reply:
Probably, yes. Its placement right now makes it basically invisible anyways. Let's hold off on removing the bottom button row for now, but we should note this in our list of things to revise for the user account refactor.

Reply:
I ended up setting the notification channel as SMS in https://github.com/ibi-group/otp-middleware/pull/83/commits/8b11107252b3b93d710eca6913d241a47456bb4a as we are now relying on the redux state fetched from the db.